### ISSUE TITLE
Am I missing some mqtt configs for home assistant?

### ISSUE THREAD

**Issue by PollieKrismis** [ParadoxAlarmInterface/pai#229]
## Alarm system
EVO48 + IP100
## Environment
Pi 3B+ (PAI docker container)
Pi 4B (HassOS/Hassio)
### Git branch
latest
## Question
Is there anything else I need to enable in the config file to get to see my mqtt messages in home assistant?

MQTT_ENABLE = True 
MQTT_HOST = ‘???.???.???.???’
MQTT_PORT = ????  
MQTT_USERNAME = ‘????’ 
MQTT_PASSWORD = ‘????’ 
MQTT_HOMEASSISTANT_DISCOVERY_PREFIX = 'homeassistant'
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE = True

I can see all the relevant EVO48 labels in the PAI log, but no failed mqtt messages.

I also cannot see any homeassistant/* messages when listening for messages in home assistant. I also do not see any failed mqtt login attempts in the home assistant logs.

Initially I had uncommented the mqtt topics below instead of the home assistant specific configs above.
MQTT_BASE_TOPIC = 'paradox'             # Root of all topics
MQTT_ZONE_TOPIC = 'zones'               # Base for zone states 
However I could also not see any paradox/* messages when listening in home assistant.

I also tried uncommenting both the topic and home assistant configs above together but still did not see any messages in home assistant.

So assuming that I've missed a step/documentation somewhere...

**Comment by yozik04** [comment]
Start fromthe wiki.

**Comment by PollieKrismis** [comment]
That was what I thought I have been doing, but find it difficult to follow as I must still learn what you have already forgotten ;-)

My goal is to simply see the state of my partitions and the zones as MQTT messages "listening" in Home Assistant MQTT broker (not yet as sensors, etc) 

I could already see the labels in the console so followed the MQTT [page](https://github.com/ParadoxAlarmInterface/pai/wiki/MQTT-Interface). I also limited items as per the configuration page so the console was easier to follow. 

Following that it mentions that I need to enable MQTT which I did. I also specified my home assistant MQTT Broker host, port, username and password. I did not see any error messages so assumed the connection could be made.

I listened for paradox/* messages in home assistant because "MQTT_BASE_TOPIC, which defaults to paradox".  As many of the items are defaulting to False when you uncomment them I assumed the rest of the MQTT documentation is only required if you wish to change the default.  

Also I did not find some of the references to items I must enable, i.e. KEEP_ALIVE_INTERFACE and MQTT_PUBLISH_STATUS and were not sure if the documentation is out of date or whether the example config file is. 

pi@pi-paradox:~/docker $ grep ALIVE pai/config/pai.conf
 KEEP_ALIVE_INTERVAL = 10   		# Interval between status updates
 MQTT_KEEPALIVE = 60     		# Keep alive
pi@pi-paradox:~/docker $ grep PUBLISH pai/config/pai.conf
 MQTT_REPUBLISH_INTERVAL = 60 * 60 * 12  # Interval for republishing all data
 MQTT_PUBLISH_DEFINITIONS = False        # Publish definitions of partitions/zones/users to MQTT. Not required in most cases.
 MQTT_PUBLISH_RAW_EVENTS = True
 MQTT_PUBLISH_COMMAND_STATUS = False  # Publish command statuses to MQTT
 MQTT_DASH_PUBLISH = False
pi@pi-paradox:~/docker $  

I could only find these references above and enabled KEEP_ALIVE_INTERVAL and MQTT_PUBLISH_COMMAND_STATUS instead. However this also did not yield any messages in the home assistant MQTT broker.

Even though it did not want to enable the alarm panel in home assistant yet, I also enabled 
MQTT_HOMEASSISTANT_DISCOVERY_PREFIX = 'homeassistant'
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE = True

listening for homeassistant/* or paradox/* still did not yield anything.

I've been over the MQTT documentation a number of times but still seem to be missing something. This is assuming I should be able to see mqtt messages in any broker it can log into rather than only working when an integration is specified.

**Comment by yozik04** [comment]
Come to gitter and send me your config privately. That will be much faster.

**Comment by yozik04** [comment]
This should be sufficient for most cases.
```
MQTT_ENABLE = True
MQTT_HOST = ‘???.???.???.???’
MQTT_PORT = ????
MQTT_USERNAME = ‘????’
MQTT_PASSWORD = ‘????’
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE = True
```

Check if you have in logs this line:
```
PAI.paradox.interfaces.mqtt.core - MQTT Broker Connected
```

If not then HOST/PORT/USERNAME/PASSWORD settings are wrong.

**Comment by PollieKrismis** [comment]
Thank you, I did not see the connected message in the log. I found the issue in the end. Turned out to be a mistake in my IP address 191 vs 192.

Funny how "blind" we get to the obvious mistakes...

Thank you for taking the time to assist.

The IP connection is very delayed and from what I read the reason why people switch to serial. Now that I got this working I can tackle the serial connection again.
